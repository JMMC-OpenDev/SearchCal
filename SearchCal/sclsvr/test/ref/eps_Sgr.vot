version https://git-lfs.github.com/spec/v1
oid sha256:1b6492d7eb1f32abe414e7d890d20149eae31610673e570ce8da553270e5865d
size 141773
