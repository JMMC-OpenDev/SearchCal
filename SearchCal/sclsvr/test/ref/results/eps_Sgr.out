version https://git-lfs.github.com/spec/v1
oid sha256:2c8ee16ed2135a48f06f9afe39ae92f5e5a8197c5da310b8ab11638a0c6a9780
size 226322
